presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -549,6 +549,13 @@ class NodeConfig : public ConfigBase {
 
   uint64_t nodeMemoryGb(
       const std::function<uint64_t()>& defaultNodeMemoryGb = nullptr) const;
+
+  /// Invoked to get the ip address of the process. In certain deployment
```

Comment:
Why we move getLocalIp from PrestoServer to configs? Meta internal will overload this method to get local ip address.